recipes/inference_labels.ipynb

```diff
@@ -4,66 +4,88 @@
    "cell_type": "markdown",
```

Comment:
<p>you can keep the paragraph together</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1051/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>